### PR TITLE
chore: bumped version in preparation for next release

### PR DESCRIPTION
**Pull request by corp-0**
I will fukken release it, yes I will.

I tested auto updating in my windows machine and it worked flawlessly (I hosted a local API and http file directory).

### CODE REVIEW

File: UnitystationLauncher/Models/ConfigFile/Config.cs [hunk 1]
```diff
@@ -10,15 +10,15 @@ namespace UnitystationLauncher.Models.ConfigFile
...
+        //Whenever you change the currentBuild here, please also update the one in UnitystationLauncher/Assets/org.unitystation.StationHub.metainfo.xml for Linux software stores. Thank you.
+        public const int CurrentBuild = 930;
 
         //file names
         private const string WinExeName = "StationHub.exe";
         private const string UnixExeName = "StationHub";
         private const string InstallationFolder = "Installations";
         public const string ApiUrl = "https://api.unitystation.org/serverlist";
-        public const string ValidateUrl = "https://api.unitystation.org/validatehubclient";
+        public const string ValidateUrl = "http://localhost:8000/";
```
Comment (RedlineTriad) [inline UnitystationLauncher/Models/ConfigFile/Config.cs]: Pretty sure this is wrong

**Review by RedlineTriad — CHANGES_REQUESTED** [review]

File: UnitystationLauncher/Models/ConfigFile/Config.cs (hunk 1 above)
Comment (corp-0) [reply UnitystationLauncher/Models/ConfigFile/Config.cs]:
```suggestion
        public const string ValidateUrl = "https://api.unitystation.org/validatehubclient";
```

Comment (corp-0) [reply UnitystationLauncher/Models/ConfigFile/Config.cs]: done

**Comment by corp-0** [comment]
https://app.codacy.com/gh/unitystation/stationhub/pullRequest?prid=8481324 codacy passed but never reported back, it seems

File: UnitystationLauncher/Assets/org.unitystation.StationHub.metainfo.xml [hunk 2]
```diff
@@ -52,5 +52,11 @@
 			<p>Stationhub is now under the MIT license. Pinging in sandbox has been fixed.</p>
 		</description>
 	</release>
+	<release version="930" date="2021-11-30">
+		<description>
+			<p>Added server validation logic that will hide servers who don't host their downloads in our CDN for security purposes.</p>
+			<p>Unitystation got a face lifting with some first iterations of future features!.</p>
```
Comment (corp-0) [inline UnitystationLauncher/Assets/org.unitystation.StationHub.metainfo.xml]:
```suggestion
			<p>StationHub got a face lifting with some first iterations of future features!.</p>
```

**Review by RedlineTriad — DISMISSED** [review]

**Comment by corp-0** [comment]
I can't merge :(

**Comment by RedlineTriad** [comment]
You could just squash and force push, that should trigger a status update? maybe?

**Comment by corp-0** [comment]
yeah, it is something else. It did the same for the previous commit, I think it simply borked

**Comment by corp-0** [comment]
fixed codacy, I'm only missing your green check now!

**Review by RedlineTriad — APPROVED** [review]

**Pull request merged**